tests/manage/pv_services/test_resource_deletion_during_pod_pvc_deletion.py

```diff
@@ -0,0 +1,320 @@
+import logging
+from concurrent.futures import ThreadPoolExecutor
+import pytest
+
+from ocs_ci.framework.testlib import ManageTest, tier4
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pvc import get_all_pvcs, delete_pvcs
+from ocs_ci.ocs.resources.pod import get_all_pods
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.utility.utils import TimeoutSampler, ceph_health_check, run_cmd
+from ocs_ci.ocs.resources.pod import (
+    get_mds_pods, get_mon_pods, get_mgr_pods, get_osd_pods
+)
+from tests import disruption_helpers
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_cephfs_storageclass, create_rbd_secret, create_cephfs_secret,
+    create_project, create_pvcs, create_pods
+)
+
+log = logging.getLogger(__name__)
+
+
+class DisruptionBase(ManageTest):
+    """
+    Base class for disruptive operations
+    """
+    num_of_pvcs = 10
+    pvc_size = '3Gi'
+    pvc_size_int = 3
+
+    def verify_resource_deletion(self, func_to_use, previous_num):
+        """
+        Wait for resource deletion to start
+
+        Args:
+            func_to_use (function): Function to be used to fetch resource info
+            previous_num (int): Previous number of resources
+
+        Returns:
+            bool: True if resource deletion has started.
+                  False in case of timeout.
+        """
+        try:
```

Comment:
we have this already in tests.manage.pv_services.test_pvc_disruptive.BaseDisruption#verify_resource_creation.
Please move this to helpers and call from both places

Reply:
Both are different actually. But possible to combine and write a separate function.